src/data/multiset/basic.lean

```diff
@@ -1778,6 +1778,22 @@ by simp_rw [←filter_eq', eq_comm]
 
 end
 
+section embedding
+
+@[simp] lemma map_le_map_iff {f : α ↪ β} {s t : multiset α} : s.map f ≤ t.map f ↔ s ≤ t :=
+⟨λ h, begin
```

Comment:
I think this would be better stated in terms of injectivity of `f` rather than requiring the bundled type.